codec/reflectcodec/type_codec.go

```diff
@@ -4,12 +4,16 @@
 package reflectcodec
 
 import (
+	"bytes"
 	"errors"
 	"fmt"
 	"math"
 	"reflect"
 
+	"golang.org/x/exp/slices"
+
 	"github.com/ava-labs/avalanchego/codec"
+	safe_math "github.com/ava-labs/avalanchego/utils/math"
 	"github.com/ava-labs/avalanchego/utils/wrappers"
 )
```

Comment:
nit: we usually put aliased imports from avalanchego at the bottom like this:

```suggestion
	"github.com/ava-labs/avalanchego/utils/wrappers"
	
	safe_math "github.com/ava-labs/avalanchego/utils/math"
)
```